lib/turn.rb

```diff
@@ -0,0 +1,17 @@
+class Turn
+    attr_reader :card, :guess
+
+    def initialize(guess, card)
+        @card = card
+        @guess = guess
+    end
+
+    def correct?
+      @guess == @card.answer
+    end
+
+    def feedback
```

Comment:
I love the whole setup of this method and how you're using implicit and explicit returns to your advantage. The only thing I would add is that because `correct?` returns a boolean you do not actually need to compare it to a boolean and you can get away with doing something like this:

```ruby
def feedback
  return "Correct!" if correct?
  "Incorrect."
end
```